index.json

```diff
@@ -92,57 +116,104 @@
     ],
     "author": [
       "Betaflight",
-      "Thomas \"BMSThomas\" Bitmatta",
-      "Zak Smiley sugarK",
-      "ctzsnooze"
+      "Directory",
+      "Ivan Efimov (Limon)",
+      "SupaflyFPV",
+      "ctzsnooze",
+      "sugarK"
     ],
     "keywords": [
       "150hz",
       "250hz",
+      "3 inch",
+      "3\"",
+      "4S",
       "5 inch",
       "5\"",
       "500hz",
       "50hz",
+      "533",
+      "5in",
+      "6 Inch",
+      "6\"",
       "6S",
+      "7 Inch",
+      "7\"",
+      "BMSThomas",
+      "CTZsoonze",
+      "Cgo",
       "Crossfire",
+      "DShot telemetry",
       "ELRS",
+      "EVO",
       "GPS",
       "HD_freestyle",
+      "HDzero",
+      "HV",
+      "IRC",
+      "KarateBrot",
+      "MultiGP",
       "OSD",
+      "RPM",
+      "SA 2.0",
+      "SA 2.0 lite",
+      "SA 2.1",
+      "Supafly",
+      "SupaflyFPV",
       "TBS",
       "adjustments",
       "aggressive",
       "aux",
+      "basic",
+      "betaflight",
       "blackbox",
       "camera",
       "camera_control",
       "cfv2",
       "cfv3",
       "cinematic",
+      "clean",
       "clear",
       "control",
+      "ctzsnooze",
+      "default",
       "defaults",
+      "divimath",
       "dynamic",
+      "expert",
       "filter",
+      "filtering",
       "filters",
       "freestyle",
       "freestyle_HD",
+      "freesytle",
       "frsky",
       "hd",
+      "hot",
       "jitter",
       "karate",
+      "karatreBrot",
       "launch",
       "launch_mode",
       "leds",
+      "limon",
       "link",
       "log",
       "logging",
       "mode",
       "modes",
       "names",
+      "nano",
+      "no rpm",
+      "noisy",
+      "normal",
+      "partial",
       "pid",
+      "pro32",
       "profile",
+      "quaddiction",
```

Comment:
Is this The RCAddict?

Reply:
same, index.json is generated automatically.
So that's again a question to @sugaarK  if hr wants to create a PR to fix that.

Reply:
Name of our local club.. it’s spelt correct